experiments/34x25/slotnick_etal_2001/uber_slotnick_pairedsquares_coordinate.m

```diff
@@ -17,3 +17,4 @@
 opts = [opts, 'force', true]; %, 'bandpass', bandpass]
 % Run sergent task by training on all images
 [trn, tst] = de_SimulatorUber('vanhateren/250', 'slotnick_etal_2001/paired-squares/coordinate', opts, args);
+tst.stats.rej.cc.anova.test.stats
```

Comment:
Doing this here will cause errors if the stats/plots change. Can we just have this dumped wherever this is computed? Then it's only in a single place, and only runs if the stat is computed.